file_app/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+# ======================================= FileData Model ============================================
+class FileData(models.Model):
+    # file_name = models.ForeignKey(FileName,on_delete=models.CASCADE,default="")
+    file_name = models.CharField(max_length=200,default="")
+    max_temp = models.IntegerField(default=0)
```

Comment:
would be better if we save data in normalized form. see and understand databases normalization topic then apply it here.